lms/static/sass/views/_program-list.scss

```diff
@@ -13,6 +13,31 @@
   @include span-columns(12);
 }
 .sidebar{
+    float: right !important;
+    //padding: $baseline;
+    margin-left: 2.35765%;
```

Comment:
Why do we need to adjust the margin-left of the sidebar? Because of the reverse flow?